.eslintrc.js

```diff
@@ -1,28 +1,36 @@
 module.exports = {
-  parser: 'babel-eslint',
-  extends: ['standard', 'standard-react', 'prettier', 'react-app'],
```

Comment:
why not just leave it as is? why do you want to change them? 🤔

Reply:
They where not working. Eslint was not shoing prettier errors. Some of the dependencies got some vulnerabilities.

Reply:
I am pretty sure it was working fine 🤔